tests/test_distributions/conftest.py

```diff
@@ -20,12 +22,17 @@ def random_samples(batch_size, num_features):
 
 @pytest.fixture()
 def diagonal_normal():
-    from bayesflow.distributions import DiagonalNormal
-
     return DiagonalNormal()
 
 
-@pytest.fixture(params=["diagonal_normal"])
+@pytest.fixture()
+def diagonal_student_t():
+    from bayesflow.distributions import DiagonalStudentT
+
+    return DiagonalStudentT()
```

Comment:
This needs an argument now. Tests should be passing otherwise.

Reply:
I am happy with the changes. Should we force a choice of df or use a default close to a Gaussian?

Reply:
I prefer forcing a choice for now. Especially a choice close to Gaussian would not be ideal I think to avoid the following scenario: A user struggles with their model and is told to try out student-t latent space. They try it out as a drop in replacement for normal (with defaults) and find no difference. Then they might think it doesn't make a difference. 

If we end up using a default later on, I would prefer it to be some small df (10 or 7 or even 3) with which we have made good experience in a bunch of experiments. For now, I would leave it without default to make the user aware of the importance of this choice. Later on we still need to document good options properly to give users a basis for decision.

Reply:
I will fix the failing test and then merge.